client/src/components/reviews/ReviewListEntry.jsx

```diff
@@ -13,9 +13,24 @@ const ReviewListEntry = (props) => {
   const bodyMaxLength = 250;
   const [extendedView, setExtendedView] = useState(false);
   const [showModal, setShowModal] = useState(false);
+  const reviewID = props.review.review_id;
 
-  const reportReview = () => {
-    console.log('Will be used with API when ready');
+  const reportReview = async (id) => {
+    try {
+      let response = await requester.reportReview(id);
```

Comment:
I believe you didn't import the Requester module. I got `ReferenceError: requester is not defined` in the client console when I tried to mark the review as helpful

Reply:
Nice catch. I think it had to do with the merging of the code last night.